### DISCUSSION TITLE
Outlook 365 PDF previewer

### DISCUSSION THREAD

**Discussion by tomerbarak01** [sumatrapdfreader/sumatrapdf#2858] [Q&A]
Hi,

Has anyone succeeded in setting Sumatra as the outlook previewer?

I have Outlook 365 (64bit) and Sumatra (64bit), but outlook still refuses to use it as the previewer.
I checked the trust center and allowed previews. I also tried to register pdfpreview.dll, but nothing seems to work.

Any help would be appreciated!

Thanks

**Comment by GitHubRulesOK** [comment]
Without 365 I cannot be sure but pre-viewer was designed for system explorer and still works even (roughly) inside sandbox and well in other explorers, However inside installed Office was a bonus depending on bitness, so you are right to use 64bit with 64bit.

365 is Office sandboxed inside a browser viewer thus there would need to be a breach of browser security to allow 365 to use external resources, I do use external SumatraPDF as a legacy browser PDF viewer but that's under my control and unlikely it would be allowed especially in Chrome/Edge etc.

I do not know how Office Online/365 could use pre-view unless it is one supplied by Acrobat extension or MS extension

**Reply by tomerbarak01** [reply]
Hi,
Thanks for your reply. 

I'll just add that I'm talking about a **local** outlook version, not an online web-based. I guess those issues would be similar with any other Outlook 2016/x64 version

![screenshot 1659873676](https://user-images.githubusercontent.com/67225737/183289541-9ff9316d-66c3-4cc2-bebe-1ee745a56fb2.jpg)

**Reply by GitHubRulesOK** [reply]
Ah so it should be working as per https://support.microsoft.com/en-us/office/outlook-pdf-preview-error-this-file-cannot-be-previewed-because-there-is-no-previewer-installed-for-it-b78bfe24-ba13-44bd-8289-1919ece7ded2 so as per part 1) do you see any message about preview not installed, as per 2) are you using any other preview handler,

However Outlook previews are not a supported feature as its system use/resources outside SumatraPDF control, it simply offers a means to be addressed, you could see if the earliest [(3,1,2)](https://www.sumatrapdfreader.org//dl//rel/3.1.2/SumatraPDF-3.1.2-64-install.exe) or latest 64 bit [3.5 prerelease](https://www.sumatrapdfreader.org/prerelease)  version installers work different but do uninstall between reinstall using the OPTIONS preview setting in installer

**Comment by Barachia** [comment]
Hi @tomerbarak01, old topic, but I found that while installing SumatraPDF, checking the box for installing for all users did the trick. By default it only installs for your personal account.
